testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/persistence/StringIndexOutOfBoundsTestCase.java

```diff
@@ -0,0 +1,91 @@
+package org.jboss.as.test.manualmode.persistence;
+
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import javax.inject.Inject;
+import org.jboss.as.test.shared.TestSuiteEnvironment;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.core.testrunner.Server;
+import org.wildfly.core.testrunner.ServerControl;
+import org.wildfly.core.testrunner.ServerController;
+import org.wildfly.core.testrunner.WildflyTestRunner;
+import org.xnio.IoUtils;
+
+/**
+ * Test if config file with wrong name can be present in 'standalone_xml_history/snapshot' folder
+
+ *
+ * @author Petr Adamec
+ */
+@ServerControl(manual = true)
+@RunWith(WildflyTestRunner.class)
+public class StringIndexOutOfBoundsTestCase {
+
+    private static final String SERVER_CONFIG = "standalone.xml";
+    private static final String BASE_PATH = TestSuiteEnvironment.getJBossHome()+File.separatorChar+"standalone"+File.separatorChar+"configuration";
+    private static final String COPY_PATH = BASE_PATH+"standalone_xml_history"+File.separatorChar+"snapshot";
+
+    @Inject
+    private ServerController container;
+
+
+    private static void copyFile(final File src, final File dst) throws IOException{
+        FileInputStream in = null;
+        FileOutputStream out = null;
+        try {
+            in = new FileInputStream(src);
+            out = new FileOutputStream(dst);
+            byte[] buffer = new byte[4096];
+            int read;
+            while ((read = in.read(buffer)) != -1) {
+                out.write(buffer, 0, read);
+            }
+        } finally {
+            IoUtils.safeClose(in);
+            IoUtils.safeClose(out);
+        }
+    }
+
+    @Test
+    public void test() throws Exception {
```

Comment:
can you include more information in test name? for instance the tested JIRA at least?